content/collections/api/en/attribution.md

```diff
@@ -16,7 +16,8 @@ summary: 'Send attribution campaign events (identified by `idfa`, `idfv`, or `ad
 ## Considerations
 
 - When Amplitude can't match attribution events to an existing user, they're held for up to 72 hours for potential user matching. If an event isn't logged for a matching user within 72 hours of receiving the attribution data, then Amplitude **drops the attribution data**.
-- For most of Amplitude's partners, attribution is matched to Amplitude users and events via the Advertising ID (IDFA, IDFV, or ADID). Therefore, you must send the Advertising ID for attribution requests and you must set the `idfa`, `idfv`, and `adid` fields in Amplitude as the Advertising ID. 
+- For most of Amplitude's partners, Amplitude uses Advertising ID (IDFA or ADID) or vendor ID (IDFV or App Set ID) for attribution to users and events. Therefore, you must send these identifiers for attribution requests and you must set the `idfa`, `idfv`, `adid` and `android_app_set_id` fields in Amplitude as the Attribution ID. 
```

Comment:
`for attribution to users and events` should be `for attribution to match users and events`?